packages/frontend/src/assets/logo/clug-logo-w.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 644.13 404.54"><defs><style>.cls-1{fill:#fff;}</style></defs><title>Fichier 2logo</title><g id="Calque_2" data-name="Calque 2"><g id="Calque_1-2" data-name="Calque 1"><path class="cls-1" d="M455.54,276.5l-58.27-1c-41.84-.06-75.87-34.81-75.87-77.5V120.5h35V198c0,23.43,18.39,42.5,41,42.5l29.73.5,11.25.2V120.5h35v156Z"/><path class="cls-1" d="M321.43,404.54l-.09-35,17.5,0c1.94,0,194.62-.5,227.54-.5a42.8,42.8,0,0,0,42.75-42.75V276.2c-11.06-.08-15.29-.24-19.16-.38-4.42-.17-8.6-.32-23.59-.32a77.75,77.75,0,0,1,0-155.5h77.75V326.25A77.84,77.84,0,0,1,566.38,404c-32.87,0-225.51.5-227.45.5ZM566.38,155a42.75,42.75,0,0,0,0,85.5c15.64,0,20.13.17,24.88.34,3.6.14,7.54.28,17.87.36V155Z"/><path class="cls-1" d="M138,241a103,103,0,0,1-1.83-206,76.59,76.59,0,0,1,53.55,22.56l24.75-24.75A111.35,111.35,0,0,0,138,0V0l-1.37,0-1.42,0V0C60.4,1.53,0,62.84,0,138c0,76.09,61.91,138,138,138h90.88V241Z"/><path class="cls-1" d="M473.38,345.5H359.88c-58.56,0-106.2-48.45-106.2-108V.5h35v237c0,40.25,31.94,73,71.2,73h113.5Z"/></g></g></svg>
```

Comment:
What is this notation ?? maybe I'm not enough a design boy to get it...

Reply:
It's a SVG file. SVG are writted in XML ^^